### ISSUE TITLE
Browser support does not seem to be correctly documented

### ISSUE THREAD

**Issue by mickaeltr** [jdalrymple/gitbeaker#679]
**Description**

The document says that the library can be used in a browser using this script:

```html
<script src="node_modules/@gitbeaker/browser/index.js" />
<script>
  const { Gitlab } = gitlab;
```

However, there is no `node_modules/@gitbeaker/browser/index.js` in the `@gitbeaker/browser` dependency.

There is a `node_modules/@gitbeaker/browser/dist/index.js` but it does not seem to export any `gitlab` object.

**Possible solutions**

I guess the documentation needs an update on this.

Thanks!

**Comment by jdalrymple** [comment]
Just added the updated docs! Tell me if you have anymore trouble :)

**Comment by jetersen** [comment]
@jdalrymple on my simple test it is undefined so definitely not resolved.

**Comment by jdalrymple** [comment]
Well thats frustrating. Ill look into it more deeply when i have a min

**Comment by jetersen** [comment]
```script
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=1, initial-scale=1.0">
  <title>Document</title>
</head>
<body>
  <script src="node_modules/@gitbeaker/browser/dist/index.js">
  </script>
  <script>
    const { Gitlab } = gitbeaker.default;
  </script>
</body>
</html>
```
You need to add `.default` at least that how it is in Chrome.

**Comment by jdalrymple** [comment]
Ugh, the default experience sucks lol

**Comment by jetersen** [comment]
I'd just like to add! Great job on splitting browser and node 👏

**Comment by mickaeltr** [comment]
Besides, it's actually the [Browser support](https://github.com/jdalrymple/gitbeaker#browser-support) that needs updates. Thanks everyone 😊

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`18.1.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/18.1.0) :rocket:
